benches/load-tester/src/runners/runner_blocks.rs

```diff
@@ -0,0 +1,51 @@
+use std::sync::Arc;
+
+use anyhow::Result;
+pub use async_nats::jetstream::consumer::DeliverPolicy;
+use fuel_streams::{client::Client, StreamConfig};
+use fuel_streams_core::prelude::*;
+use futures::StreamExt;
+
+use super::results::LoadTestTracker;
+
+pub async fn run_blocks_consumer(
+    client: &Client,
+    load_test_tracker: Arc<LoadTestTracker>,
+) -> Result<()> {
+    // Create a new stream for blocks
+    let stream = fuel_streams::Stream::<Block>::new(client).await;
+
+    // Configure the stream to start from the last published block
+    let config = StreamConfig {
+        deliver_policy: DeliverPolicy::Last,
+    };
+
+    // Subscribe to the block stream with the specified configuration
+    let mut sub = stream.subscribe_with_config(config).await?;
+
+    // Process incoming blocks
+    while let Some(bytes) = sub.next().await {
+        match bytes.as_ref() {
+            Err(_) => load_test_tracker.increment_error_count(),
+            Ok(message) => {
+                load_test_tracker.increment_message_count();
+                let decoded_msg =
+                    Block::decode_raw(message.payload.to_vec()).await;
```

Comment:
Decoding the actual type shouldn't be a concern in these load benches

Reply:
The only reason I am decoding it back is to retrieve the timestamp of publishing of the StreamData as I am later doing some stats based on that.

Reply:
It has no effect on the load testing itself.

Reply:
Okay. But it would be nice to have a single runner that accepts a `impl Streamable` stream argument to dynamically compute these. Repeating the same logic per stream type makes it seem as though the load testing has some biases based on the stream type

Reply:
done